docs/models/noddi.rst

```diff
@@ -1,14 +1,130 @@
+
+**Childhood and Adolescence**: 
+Neurite density continues to rise through childhood and adolescence, though the rate slows with age. 
+Multiple studies (e.g. :cite:t:`Genc2017NDIAgeDevelopment`, :cite:t:`Mah2017NODDIChildDevelopment`) 
+found strong positive correlations between age and ICVF/NDI in children.
+By contrast, orientation dispersion (ODI) remains relatively stable during childhood,
+indicating that fiber organization (coherence) does not markedly increase after early childhood in most regions :cite:p:`Mah2017NODDIChildDevelopment`.
+The net effect is that white matter FA increases in youth are chiefly driven by increasing neurite density (more and thicker axons with more myelin),
+rather than fibers becoming straighter or more aligned. 
+Children with higher NDI tend to have higher FA even if ODI is unchanged, 
+meaning microstructural density is a key determinant of developmental differences :cite:p:`Mah2017NODDIChildDevelopment`.
+
+**Late Adolescence to Early Adulthood**: 
+Neurite density in many tracts appears to plateau by the early 20s, 
+reaching peak or near-peak values, while ODI might begin to creep upward. 
+:cite:t:`Chang2015NODDIDataset` noted that NDI followed a logarithmic growth curve that leveled off in the 20s-30s, 
+whereas ODI began an exponential upward trend in the 30s.
+This implies that during the transition to adulthood, continued incremental myelination is eventually counteracted by emerging microscopic disorganization 
+(potentially reflecting early branching/pruning or cumulative minor damage), 
+which foreshadows the patterns seen in later aging.
+
+**Mid-Life and Healthy Aging**: 
+In mid-to-late adulthood, NODDI reveals progressive loss of neurite density and increasing fiber dispersion. 
+Large-scale data from the UK Biobank showed that older age is associated with lower ICVF 
+and higher ODI across virtually all white matter tracts :cite:p:`Lawrence2021UKBiobankWhiteMatter`. 
+Notably, these microstructural changes can be detected even in healthy adults with no disease, reflecting normative brain aging. 
+Age-related ODI increases often accelerate in the later decades, 
+consistent with accumulating structural disintegration.
+:cite:t:`Lawrence2021UKBiobankWhiteMatter` also found that CSF fraction (ISOVF) 
+increases with age, indicating expanding extracellular space.
+
+**Clinical and Cognitive Relevance**: 
+The lifespan changes in NODDI metrics align with known windows of brain plasticity and decline. 
+The steep NDI increase in childhood corresponds to learning and cognitive development, 
+whereas rising ODI in late life correlates with cognitive slowing and increased white matter vulnerability. 
+Importantly, NDI has been found to be a better predictor of chronological age than standard DTI measures in youth :cite:p:`Genc2017NDIAgeDevelopment`.
+
+
+NODDI Methodological Warnings and Caveats
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Model Assumptions and Biases**: 
```

Comment:
https://doi.org/10.1371/journal.pone.0217118 is probably a better citation here than Zhao. Zhao is more applicable to the childhood/adolescence trajectories.